### PR TITLE
Git Pages

### PR DESCRIPTION
**Pull request by jdebacker**
This PR adds the `.nojekyll` file to try to build sphinx docs for github pages.  Also updates links for the PSL catalog.

### CODE REVIEW

File: PSL_catalog.json [hunk 1]
```diff
@@ -113,22 +113,23 @@
     },
     "link_to_webapp": {
         "data": null,
-        "source": null,
+        "source": "https://compute.studio/PSLmodels/OG-USA/",
         "type": null,
         "start_header": null,
         "end_header": null
+        "data": "<a href=\"https://compute.studio/PSLmodels/OG-USA/\">Link to webapp</a>"
```
Comment (rickecon) [inline PSL_catalog.json:120]: @jdebacker . I looked at the source file for this line, and I can't figure out why the colon is highlighted in red. Any idea what is going on here?

**Comment by rickecon** [comment]
@jdebacker. It looks like once the budget window problems in the tests with PR #582 are solved, then this PR will solve.

**Comment by jdebacker** [comment]
@rickecon Yes - I made this branch off the `start2021` branch.  I'm updating the TPI test files for the 2021 start year now and will commit them to PR #582 soon.  Let's merge that PR before this one.

File: PSL_catalog.json (hunk 1 above)
Comment (jdebacker) [reply PSL_catalog.json:120]: @rickecon  - missing comma.  Thanks for noticing.  Just fixed.

**Comment by jdebacker** [comment]
@Peter-Metz do the updates to `PSL_catalog.json` look good to you?

File: PSL_catalog.json [hunk 2]
```diff
@@ -113,22 +113,23 @@
     },
     "link_to_webapp": {
         "data": null,
```
Comment (Peter-Metz) [inline PSL_catalog.json:115]: @jdebacker delete this line since you added the webapp url below

**Comment by Peter-Metz** [comment]
@jdebacker Just left a minor comment. I've been catching these formatting issues with this [online JSON validator](https://jsonlint.com/)

**Comment by jdebacker** [comment]
@Peter-Metz I made that fix - thanks!

Also, installed `jsonlint` package for Atom, but it wasn't catching that extra data element - though the online one did.  Be nice to have that validation in my text editor.

**Comment by jdebacker** [comment]
FYI, @Peter-Metz the JSON linter for VS Code has better functionality.  Time for me to completely switch over.

**Comment by Peter-Metz** [comment]
@jdebacker Ah, good to know. I've been using Sublime and the JSON linter plugin isn't catching duplicate keys either. An editor switch might be coming on this end too...

**Pull request merged**